_data/simple-icons.json

```diff
@@ -8415,7 +8446,10 @@
         {
             "title": "Spacemacs",
             "hex": "9266CC",
-            "source": "http://spacemacs.org/"
+            "source": "http://spacemacs.org/",
```

Comment:
Nice find on https://github.com/nashamri/spacemacs-logo!

Did you manage to verify that is indeed the origin of the logo? Would it make sense to update the source to this URL? Did you compare the SVG in that repository to the one we have?

```suggestion
            "source": "https://github.com/nashamri/spacemacs-logo",
```

Reply:
His name is mentioned in the attribution, so I though it might be the origin.
I kept the source as-is, since our version match the logo version found in the website footer.
![image](https://user-images.githubusercontent.com/12817388/126814529-687d4543-f3a0-4aee-8f0a-c37301cb255a.png)

Reply:
That is pretty convincing yes, thanks for clarifying.

As for updating the source, based on that I agree with you that we should keep the source as is :+1: